pandas/tseries/offsets.py

```diff
@@ -1226,7 +1226,8 @@ def _get_roll(self, i, before_day_of_month, after_day_of_month):
         return roll
 
     def _apply_index_days(self, i, roll):
-        i += (roll % 2) * Timedelta(days=self.day_of_month).value
+        nanos = (roll % 2) * Timedelta(days=self.day_of_month).value
```

Comment:
leftover from another PR (and I had commented on that), remove from here.

Reply:
You’re rIght that this is duplicated, but it is correct in both places, and this PR breaks without it.

Reply:
see other PR, this needs a doc-string